### ISSUE TITLE
Bug: It doesn't work on MIUI 14 - Android 13

### ISSUE THREAD

**Issue by TheKerevas** [d4rken-org/sdmaid#5911]
### What happened?
On my POCO F4 GT with MIUI 14 and Android 13 it doesn't remove the files it finds, and clearing the cache with accessibility mode doesn't work 

### What did you expect to happen?
I expected the file cleanup work

### How can I reproduce the issue?
Just start file scans and try to delete them, even clearing the cache with accessibility 

### SD Maid
Version: `5.5.8 (50508)`
Pro: `isPro=true || isComplete=true (unl=true, iap=false, acc=false)`

### Device
Root: `RootContext(rootState=Root(state=UNAVAILABLE)`
Fingerprint: `POCO/ingres_eea/ingres:13/TKQ1.220807.001/V14.0.1.0.TLJEUXM:user/release-keys`

**Comment by d4rken** [comment]
Can you record a log of the accessibility service issue?
https://github.com/d4rken-org/sdmaid/wiki/Reporting-a-bug#debug-log

**Comment by TheKerevas** [comment]
> Can you record a log of the accessibility service issue? https://github.com/d4rken-org/sdmaid/wiki/Reporting-a-bug#debug-log

Sunday I try and send the log

**Comment by TheKerevas** [comment]
> Can you record a log of the accessibility service issue? https://github.com/d4rken-org/sdmaid/wiki/Reporting-a-bug#debug-log

Ok I recorded the log of the whole cleaning process, including the "Application Cleaning" one through the accessibility service. The log was up to trying to clean 2 apps with the accessibility service, I even made a video to show what happens while it tries to clean. Before with MIUI 13 and Android 12 I saw that it cleared the cache for each application and much faster, now it no longer erases anything and it lingers for each application for a long time

[sdmaid_logfile_1676212187740.txt.zip](https://github.com/d4rken-org/sdmaid/files/10716281/sdmaid_logfile_1676212187740.txt.zip)


https://user-images.githubusercontent.com/101466273/218317354-c58c1075-e7cf-4c9d-83cc-ab48e6c92672.mp4

**Comment by Malgotra** [comment]
Same thing happening in MIUI 14 Android 12. Accessibility service stuck.

**Comment by d4rken** [comment]
The logs show the cause (no UI data from system), but don't point to a solution.
I'll have to do trial & error, but I don't have a MIUI 14 device.
I'm looking for a cheap one to buy currently (broken display or something).

What's the cheapest device running MIUI 14 that is easy to flash and root?

**Comment by TheKerevas** [comment]
> The logs show the cause (no UI data from system), but don't point to a solution. I'll have to do trial & error, but I don't have a MIUI 14 device. I'm looking for a cheap one to buy currently (broken display or something).
> 
> What's the cheapest device running MIUI 14 that is easy to flash and root?

I'm looking around, but the least expensive with MIUI 14 currently cost around €300/350 new

**Comment by Malgotra** [comment]
My device is Redmi note 10 which is around 140 euros.

**Comment by TheKerevas** [comment]
> My device is Redmi note 10 which is approx. €140 according to Indian currency.

Does it have MIUI 14?

**Comment by Malgotra** [comment]
> 

Yes. Got 2 days ago in India.

**Comment by Malgotra** [comment]
> > My device is Redmi note 10 which is approx. €140 according to Indian currency.
> 
> Does it have MIUI 14?

Yes. Got 2 days ago in India.

**Comment by Pil0tXia** [comment]
Redmi K40 MIUI 14.0.4 has this bug too.

**Comment by AlanTuringTester** [comment]
Redmi K50 MIUI 14.0.4 has this bug too.

**Comment by chelseaislan** [comment]
My [Xiaomi 12T](https://www.mi.com/global/product/xiaomi-12t/) which has got an update to Android 13 with MIUI 14 skin has this bug too.

[sdmaid_logfile_1677504353814.txt.EN.zip](https://github.com/d4rken-org/sdmaid/files/10840106/sdmaid_logfile_1677504353814.txt.EN.zip)

**Comment by cerberus6666** [comment]
would like to also vouch the 12T..from what i can see it gets stuck on appcleaner as the UI looks a tad smaller and the SDmaid is made for a certain size UI which has been changed on the update

**Comment by d4rken** [comment]
Please report back after beta v5.5.9

**Comment by Malgotra** [comment]
MIUI 14 Android 12. Fixed.

**Comment by chelseaislan** [comment]
Fixed in MIUI 14 (Android 13).

**Comment by cerberus6666** [comment]
works perfect the beta good job @d4rken !
